app/frontend/src/features/NotFoundPage.tsx

```diff
@@ -1,10 +1,12 @@
-import { Box } from "@material-ui/core";
+import { Box,Link as MuiLink } from "@material-ui/core";
```

Comment:
run `yarn format`

Reply:
I ran yarn format on my local machine and added the changes to the file. Let me know if anything else should be changed! Thank you!